qiskit_ibm/runtime/runtime_job.py

```diff
@@ -385,6 +388,15 @@ def backend(self) -> Backend:
         """
         return self._backend
 
+    def runtime(self) -> str:
+        """Return the runtime image used for the job.
```

Comment:
During the runtime workshop someone asked why some of the runtime job attributes are properties (like `inputs` and some are methods (like `job_id()`). The reason `job_id()` and `backend()` are methods is to match that of `IBMQJob`, and at the time I didn't think it was necessary to make the others methods. However, we should either 1) continue making everything other than those 2 properties or 2) make them all methods

@rathishcholarajan thoughts on this?

Reply:
It felt odd to me when I use these as methods. I prefer them to be properties. Maybe we can change `job_id` and `backend` to properties just for runtime? I don’t care if change for IBMQJob or not.

Reply:
We can't change `job_id` and `backend` for IBMQJob because they're inherited from `Job` (base class in Terra for all provider jobs). I was hoping for some consistency with IBMQJob, which has mostly just methods, but perhaps IBMRuntimeJob is different enough we can do just properties. Making `job_id` and `backend` properties would need to be done as part of 0.1 though since it's a breaking change.

Reply:
I've created an issue https://github.com/Qiskit-Partners/qiskit-ibm/issues/87